content/en/tracing/trace_collection/automatic_instrumentation/single-step-apm.md

```diff
@@ -234,20 +234,19 @@ To enable single step instrumentation with Helm:
 
 You can choose to selectively instrument specific namespaces or choose to not instrument them.
 
-To enable instrumentation for specific namespaces, replace `enabled: true` with `enabledNamespaces` configuration in your `datadog-values.yaml` file:
+To enable instrumentation for specific namespaces, add `enabledNamespaces` configuration to your `datadog-values.yaml` file:
 {{< highlight yaml "hl_lines=6-8" >}}
```

Comment:
```suggestion
{{< highlight yaml "hl_lines=6-9" >}}
```
^ Update highlight since we added a line.

Reply:
done